cfg/1.13/node.yaml

```diff
@@ -222,6 +222,8 @@ groups:
   - id: 2.1.11
     text: "Ensure that the --cadvisor-port argument is set to 0 (Scored)"

```

Comment:
As above, it would be good to add [DEPRECATED] into the text description to match the benchmark